js/HelpTextComponent.js

```diff
@@ -156,4 +167,17 @@ HelpTextComponent.prototype.handleFocusOut = function (e) {
     }
 }
 
+/**
+ * Toggle focusability of elements
+ */
+HelpTextComponent.prototype.toggleChildElementInteractivity = function ($container, interactive) {
+    var focusableElements = 'a[href], button';
+
+    if (interactive === true) {
```

Comment:
should work with just `if (interactive) {` ?

Reply:
In this instance yeah, it would. I prefer to be more explicit though as it's possible the value of that var might change in the future.